src/Pagination/Cursor.php

```diff
@@ -66,10 +61,9 @@ public function __construct($current = null, $prev = null, $next = null, $count
     }
 
     /**
-     * Get the current cursor value.
-     *
-     * @return mixed
+     * {@inheritDoc}
      */
+    #[\ReturnTypeWillChange]
```

Comment:
Attributes are php8.0 feature and this particular attribute is php 8.1, but codebase minimal version is **7.4**.

Don't test crash on this line at 7.4?

https://php.watch/versions/8.1/ReturnTypeWillChange

Reply:
Attributes are comments in every version before 8.0, and in every other version they are not evaluated at runtime. 

The only way this could be an issue is if someone got the attribute from the method reflection instance and tried to run `$attribute->newInstance();` outside of a try/catch which is pretty unlikely since it has no reason to be instantiated in any version.